app/controllers/api/teams_controller.rb

```diff
@@ -3,7 +3,9 @@ class TeamsController < ApplicationController
     before_action :set_team, only: %i[show update destroy]
 
     def index
-      teams = Team.order(created_at: :asc)
+      teams = Team.order(created_at: :asc).map do |team|
+        { team: team, results: team.results }
+      end
```

Comment:
This causes N+1 query, then try this:

```rb
      teams = Team.order(created_at: :asc).includes(:results).map do |team|
        { team: team, results: team.results }
      end
```

or using `#as_json` w/ `:only` option is also okay,

```suggestion
      teams = Team.order(created_at: :asc).includes(:results).as_json(
        only: %i(id name founded_year league ...),
        include: { results: { only: %i(id team_id ...) },
      )
```

You can omit `:only` options, but it is dangerous when you add a new column and don't wanna expose to the client. (like confidential one)

Reply:
I fixed it, thank you. f0e2252252ab9a72ec59f8a64e06532d12a71bc6